definitions/vertical_scaling.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Vertical Scaling
+status: feedback appreciated
+category: concept
+---
+## Canary Deployment
+
+### What it is
+
```

Comment:
Is your intent to leave this blank? That's ok if it is, we just need to know your intent.